src/components/DateRangePicker/index.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import 'test-util/setup';
+import { mount } from 'enzyme';
+import DateRangeInput from './index';
+
+const setup = (props = {}) => {
+  const testProps = { ...props };
+  return mount(<DateRangeInput {...testProps} />);
+};
+
+describe('render', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = setup();
+  });
+
+  it('without errors', () => {
+    expect(wrapper.length).toBe(1);
+  });
+
+  it('startDatePlaceholderText prop`s default value is `Start Date`', () => {
+    const { startDatePlaceholderText } = wrapper.props();
+    expect(startDatePlaceholderText).toBe('Start Date');
```

Comment:
i think youre missing the point of the test - don't test implementation

Reply:
1. most importantly: see if you cant enter a date value into the field using 'simulate.change()' and see if the onDataChanged triggers
2. secondly: check that passed in props are represented and reflected in the component

Reply:
done